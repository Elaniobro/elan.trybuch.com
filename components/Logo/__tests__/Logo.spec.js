import { shallowMount } from '@vue/test-utils'
import Logo from '@/components/Logo/Logo'

let wrapper
beforeEach(() => {
  wrapper = shallowMount(Logo, {})
})

afterEach(() => {
  wrapper.destroy()
})

describe('Logo.vue', () => {
  test('is a Vue instance', () => {
    expect(wrapper).toBeTruthy()
  })

  test('matches its snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  test('HTML matches its snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('renders a single image', () => {
    expect(wrapper.findAll('svg').length).toEqual(1)
  })
})
