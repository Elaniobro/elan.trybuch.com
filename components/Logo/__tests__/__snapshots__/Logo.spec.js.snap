// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.vue HTML matches its snapshot 1`] = `
<div class="logo__wrapper"><svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 178 79" class="logo__svg">
    <defs></defs>
    <path d="M27 9.4C32.2 16 39 28 39 30.5c0 1.7-1.5 2.6-5.4 3.5-4.8.9-4 3.5.9 3.3l4-.1v6.6c0 3.7.4 7.9.9 9.4.7 2.4.5 2.9-1.8 3.9-6.5 3-18.2-5.3-23.6-16.7-1.7-3.4-3-6.7-3-7.4 0-1.3 7.4 2.7 13.1 7.2 3.5 2.6 7.9 3.8 7.9 2 0-.5-1.2-1.9-2.6-3.3-2.9-2.7-17.8-21.9-17-21.9.3 0 3.8 2.9 7.8 6.5s8.1 6.5 9.1 6.5c2.5 0 1.3-1.7-5.8-8.1C5.7 5.9 2 3.3 2 7.1 2 8.3 11.1 22 15.8 27.9c2 2.4 3.2 4.2 2.6 3.9-12.3-5.9-13.7-4.6-8.8 7.8 3.4 8.4 11.6 17.6 17.9 20 4.5 1.7 4.9 1.7 18.4-.4 6.7-1.1 9.3-3.8 15.3-16l4.1-8.2 1.6 4.7c1.7 4.9 3.9 7 6.3 6 1.5-.5 3.4-12.5 4.1-25.2l.2-5 3.4 9c4.2 11.3 6.3 15.5 11.4 23 18 26.9 37.1 35.3 64.2 28.5 9.8-2.4 16.1-5.2 13.4-5.7-.9-.2-4.9.6-9 1.7-4.1 1.2-11.5 2.4-16.5 2.7-16.7 1.1-28.8-4.8-41.7-20.2-7.9-9.4-12.1-16.6-17.8-30.3-5.2-12.4-6.7-14.6-9.1-13.7-1.1.5-1.7 3.1-2.2 11.3-.4 5.9-1.1 12.7-1.5 15.2l-.7 4.5-2.3-7.3C67.4 28.5 66.5 27 65 27c-1.1 0-2 .7-2 1.5 0 3.3-1.7 1.2-3.4-4.3-1.9-6.1-4.1-10.2-5.5-10.2-.5 0-3.2 2.9-6 6.5S42.6 27 42.2 27c-.4 0-2.3-3-4.2-6.8C34.6 13.5 25.5 3 23.2 3c-.7 0 1 2.9 3.8 6.4zm28.9 14.5c.6 1.9 1 3.6.8 3.7-.1.2-1.9-.3-4-1l-3.6-1.2 2.2-3.2c2.4-3.4 2.9-3.3 4.6 1.7zm.4 8.6c2 1.9 3.7 4 3.7 4.5 0 3.6-10.6 19-13.1 19-.4 0-.5-3-.3-6.8.4-4.5.1-8.3-1-11.6-1.9-6.2-.9-8.6 3.7-8.6 2.4 0 4.3 1 7 3.5zM42.9 45.7c-.1 5.4-.2 5.5-.9 2.3-.8-3.1-.4-8 .6-8 .2 0 .4 2.6.3 5.7z"></path>
  </svg></div>
`;

exports[`Logo.vue matches its snapshot 1`] = `
<div class="logo__wrapper"><svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 178 79" class="logo__svg">
    <defs></defs>
    <path d="M27 9.4C32.2 16 39 28 39 30.5c0 1.7-1.5 2.6-5.4 3.5-4.8.9-4 3.5.9 3.3l4-.1v6.6c0 3.7.4 7.9.9 9.4.7 2.4.5 2.9-1.8 3.9-6.5 3-18.2-5.3-23.6-16.7-1.7-3.4-3-6.7-3-7.4 0-1.3 7.4 2.7 13.1 7.2 3.5 2.6 7.9 3.8 7.9 2 0-.5-1.2-1.9-2.6-3.3-2.9-2.7-17.8-21.9-17-21.9.3 0 3.8 2.9 7.8 6.5s8.1 6.5 9.1 6.5c2.5 0 1.3-1.7-5.8-8.1C5.7 5.9 2 3.3 2 7.1 2 8.3 11.1 22 15.8 27.9c2 2.4 3.2 4.2 2.6 3.9-12.3-5.9-13.7-4.6-8.8 7.8 3.4 8.4 11.6 17.6 17.9 20 4.5 1.7 4.9 1.7 18.4-.4 6.7-1.1 9.3-3.8 15.3-16l4.1-8.2 1.6 4.7c1.7 4.9 3.9 7 6.3 6 1.5-.5 3.4-12.5 4.1-25.2l.2-5 3.4 9c4.2 11.3 6.3 15.5 11.4 23 18 26.9 37.1 35.3 64.2 28.5 9.8-2.4 16.1-5.2 13.4-5.7-.9-.2-4.9.6-9 1.7-4.1 1.2-11.5 2.4-16.5 2.7-16.7 1.1-28.8-4.8-41.7-20.2-7.9-9.4-12.1-16.6-17.8-30.3-5.2-12.4-6.7-14.6-9.1-13.7-1.1.5-1.7 3.1-2.2 11.3-.4 5.9-1.1 12.7-1.5 15.2l-.7 4.5-2.3-7.3C67.4 28.5 66.5 27 65 27c-1.1 0-2 .7-2 1.5 0 3.3-1.7 1.2-3.4-4.3-1.9-6.1-4.1-10.2-5.5-10.2-.5 0-3.2 2.9-6 6.5S42.6 27 42.2 27c-.4 0-2.3-3-4.2-6.8C34.6 13.5 25.5 3 23.2 3c-.7 0 1 2.9 3.8 6.4zm28.9 14.5c.6 1.9 1 3.6.8 3.7-.1.2-1.9-.3-4-1l-3.6-1.2 2.2-3.2c2.4-3.4 2.9-3.3 4.6 1.7zm.4 8.6c2 1.9 3.7 4 3.7 4.5 0 3.6-10.6 19-13.1 19-.4 0-.5-3-.3-6.8.4-4.5.1-8.3-1-11.6-1.9-6.2-.9-8.6 3.7-8.6 2.4 0 4.3 1 7 3.5zM42.9 45.7c-.1 5.4-.2 5.5-.9 2.3-.8-3.1-.4-8 .6-8 .2 0 .4 2.6.3 5.7z"></path>
  </svg></div>
`;
